consensus/phases/communicator.go

```diff
@@ -218,3 +268,10 @@ func (nc *NaiveCommunicator) phase2DataHandler(request network.Request) {
 		return
 	}
 }
+
+func (nc *NaiveCommunicator) phase3DataHandler(request network.Request) {
+	if request.GetType() != types.Phase3 {
```

Comment:
why we don't collect results in handler?

Reply:
forgot. done